src/actions/WindowActions.js

```diff
@@ -761,6 +766,113 @@ export function fetchTopActions(windowType, docId, tabId) {
   };
 }
 
+const getAPIUrl = function({ windowId, docId, tabId, rowId, path }) {
```

Comment:
If it's just an API call, why not move it to `/api` ?

Reply:
Moved